packages/babel-traverse/src/scope/lib/renamer.js

```diff
@@ -20,7 +20,7 @@ const renameVisitor = {
     }
   },
 
-  "AssignmentExpression|Declaration"(path, state) {
+  "AssignmentExpression|Declaration|VariableDeclarator"(path, state) {
```

Comment:
```suggestion
  "AssignmentExpression|Declaration|VariableDeclarator"(path, state) {
    if (path.isVariableDeclaration()) return;
```

Otherwise we check var bindings twice. It shouldn't cause problems, but it's unnecessary.